### DISCUSSION TITLE
Druid Datasources dashboard console not working

### DISCUSSION THREAD

**Discussion by kamal-singhal** [Sunbird-Obsrv/Community#79] [Issues]
Hi team,

Druid Datasources dashboard console not working. we checked the druid coordinator, router, overload service is working fine but on datasource dashboard not showing the data and it's give below error:
**request failed with status code 500**  
kindly help us and let us know the solution steps.
for your reference, screen shot has been attached. 

![druid dashboard](https://github.com/Sunbird-Obsrv/Community/assets/125978084/bb9be4d6-7e8e-413d-a4f2-d314397b411e)

**Comment by manjudr** [comment]
@kamal-singhal  - Are you still encountering the same issue? Kindly review all the service logs and provide details about the exception you are encountering. If any of the services are in an unhealthy state, it is likely the cause of the issue you're facing.
